packages/components/time_region_value/src/node_modules/components/SettingsRow.svelte

```diff
@@ -0,0 +1,113 @@
+<script>
+	import Icon from '@svizzle/ui/src/icons/Icon.svelte';
+	import ChevronDown from '@svizzle/ui/src/icons/feather/ChevronDown.svelte';
+	import ChevronUp from '@svizzle/ui/src/icons/feather/ChevronUp.svelte';
+	import Globe from '@svizzle/ui/src/icons/feather/Globe.svelte';
+	import Switch from '@svizzle/ui/src/Switch.svelte';
+
+	import sharedTheme from 'theme';
+
+	const defaultTheme = {
+		colorRef: sharedTheme.colorRef,
+		colorSelected: sharedTheme.colorSelected,
+	};
+	const defaultHandlers = {
+		toggledColorScheme: null,
+		toggledFiltering: null,
+		toggledGeoModal: null,
+		toggledRanking: null,
+	}
+
+	export let colorScheme = {
+		value: 'Red-Blue',
+		values: ['Red-Blue', 'Green-Blue']
+	}
+	export let flags = {
+		areThereUnselectedNUTS1Regions: false,
+		doFilter: false,
+		isGeoModalVisible: false,
+		showRankingControl: false,
+	}
+	export let handlers = defaultHandlers;
+	export let theme = {
+		colorRef: defaultTheme.colorRef,
+		colorSelected: defaultTheme.colorSelected,
+	}
+
+	$: handlers = handlers ? {...defaultHandlers, ...handlers} : defaultHandlers;
+	$: theme = theme ? {...defaultTheme, ...theme} : defaultTheme;
+	$: globeStroke =
+		flags.areThereUnselectedNUTS1Regions ? theme.colorSelected : theme.colorRef;
```

Comment:
Formatting. Maybe this is preferred?:
```
	$: handlers = handlers
		? {...defaultHandlers, ...handlers}
		: defaultHandlers;
	$: theme = theme ? {...defaultTheme, ...theme} : defaultTheme;
	$: globeStroke = flags.areThereUnselectedNUTS1Regions
		? theme.colorSelected :
		theme.colorRef;
```

Reply:
Here too there lines are shortes than 80 chars. Agree the `globeStroke` line isn't very readable, I'll rename `areThereUnselectedNUTS1Regions` to `noRegionsSelected`